### ISSUE TITLE
Publish to Pypi

### ISSUE THREAD

**Issue by listingmirror** [fulfilio/python-magento#11]
Is this on Pypi?  I know the old python-magento is, but not seeing this one...   It would be helpful to do so...

**Comment by adamlwgriffiths** [comment]
```
$ pip show magento
---
Metadata-Version: 1.1
Name: magento
Version: 2.2
Summary: Magento Core API Client
Home-page: https://github.com/fulfilio/magento/
```

**Comment by tarunbhardwaj** [comment]
Released version 3.0
